templates/layout.html

```diff
@@ -94,7 +103,7 @@
 		}
 		
 		$(document).ready(function() {
-
+			getHx_name();
```

Comment:
Change this to:
`
$('#hx_name-placeholder').text({{session['ht_hx_name']}});
`